### PR TITLE
Decompiled various small files

### PR DESCRIPTION
**Pull request by Random06457**
- Decompiled `code_8006C360.c`
- Decompiled `prenmi_buf.c` (previously `code_8007BE60.c`)
- Decompiled `code_8007BF10.c`
- Documented `sys_ucode.c`

### CODE REVIEW

File: src/code/code_8006C360.c [hunk 1]
```diff
@@ -1,5 +1,80 @@
...
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8006C360/func_8006C360.s")
+//#pragma GLOBAL_ASM("asm/non_matchings/code/code_8006C360/func_8006C360.s")
 
+
+typedef struct
+{
+    u16 data[9];
+} struct_8011F3F0; // size = 0x12
```
Comment (Roman971) [inline src/code/code_8006C360.c]: Can this not just be a 2D array?

File: src/code/code_8006C360.c [hunk 2]
```diff
@@ -1,5 +1,80 @@
 #include <ultra64.h>
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8006C360/func_8006C360.s")
+//#pragma GLOBAL_ASM("asm/non_matchings/code/code_8006C360/func_8006C360.s")
```
Comment (Roman971) [inline src/code/code_8006C360.c]:
```suggestion
```

**Comment by Random06457** [comment]
I documented `z_msgevent.c` but I think I'll leave `code_8006C360.c` to someone else since I currently don't know a lot about NPCs

**Comment by Roman971** [comment]
This is minor but based on its location I think `prenmi_buff.c` should be a z_ file and thus be called `z_prenmi_buff.c`.

**Pull request merged**